src/assets/i18n/ru.json

```diff
@@ -0,0 +1,68 @@
+{
+  "ADVANCED": "Дополнительные параметры",
```

Comment:
I'll do native translations for it after.
and btw what is your native language?